src/OGL-UK-2.0.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+    <license isOsiApproved="false" licenseId="OGL-UK-2.0" name="Open Government License (UK) v2.0" listVersionAdded="3.3">
```

Comment:
The name tag in line 3 says "Open Government License (UK) v2.0" -- looks like it should be "Licence" with a "c".